README.md

```diff
@@ -64,19 +70,27 @@ We're following the general Ansible contributor guidelines; see [Ansible Communi
 If you want to clone this repositority (or a fork of it) to improve it, you can proceed as follows:
 1. Create a directory `ansible_collections/community`;
 2. In there, checkout this repository (or a fork) as `crypto`;
-3. Add the directory containing `ansible_collections` to your [ANSIBLE_COLLECTIONS_PATHS](https://docs.ansible.com/ansible/latest/reference_appendices/config.html#collections-paths).
+3. Add the directory containing `ansible_collections` to your [ANSIBLE_COLLECTIONS_PATH](https://docs.ansible.com/ansible/latest/reference_appendices/config.html#collections-paths).
+
+See [Ansible's dev guide](https://docs.ansible.com/ansible/devel/dev_guide/developing_collections.html#contributing-to-collections) for more information.
 
 ## Release notes
 
-<!--Add a link to a changelog.md file or an external docsite to cover this information. -->
+See the [changelog](https://github.com/ansible-collections/collection_template/blob/master/CHANGELOG.rst).
```

Comment:
dead link

Reply:
It will exist once the first release is created :)

Reply:
In collection_template?

Reply:
LOL, great catch! I really wonder where I copied that from...